src/controllers/PenaltyDetailsController.ts

```diff
@@ -55,7 +58,12 @@ export class PenaltyDetailsController extends BaseAsyncHttpController {
         const extraData = session.getExtraData();
 
         const changePenaltyIdentifier = (appeals: Appeal) => {
-            appeals[AppealKeys.PENALTY_IDENTIFIER] = body;
+            const companyNumber = appeals[AppealKeys.PENALTY_IDENTIFIER][PenaltyIdentifierKeys.COMPANY_NUMBER];
```

Comment:
is this working for you? the company number is not being sanitised for me

Reply:
I thought this was working. I just made this pass the linting test, I don't think I changed the logic though...

Reply:
you didn't :-) i'll take another look